test/utils.js

```diff
@@ -0,0 +1,48 @@
+var assert = require('chai').assert;
+var sass = require('node-sass');
+
+var render = function(options) {
+  options.outputStyle = 'compressed';
```

Comment:
String comparisons are hard and messy against non compressed out. Testing
output mode is testing the node-sass implementation and out of scope.
On 17 May 2016 1:15 AM, "Hugo Giraudel" <notifications@github.com> wrote:

> In test/utils.js
> <https://github.com/kaelig/sass-project-boilerplate/pull/1#discussion_r63369954>
> :
>
> > @@ -0,0 +1,48 @@
> > +var assert = require('chai').assert;
> > +var sass = require('node-sass');
> > +
> > +var render = function(options) {
> > +  options.outputStyle = 'compressed';
>
> Any reason you hard-code this rather than respecting possibly given
> outputStyle?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/kaelig/sass-project-boilerplate/pull/1/files/4c6935f6f309b5617eadc1d7c7f49d23f64a9ede#r63369954>
>